### PR TITLE
Using stdin instead of dialoguer for prompts

### PR DESCRIPTION
**Pull request by ghost**
### Minor Changes
- For inputting and editing, use `stdin` instead of `dialoguer`.

### CODE REVIEW

File: src/note.rs [hunk 1]
```diff
@@ -39,10 +40,10 @@ impl Note {
 
         cursor_to_origin()?;
         println!(
-        "If you're done inputting a field, you can press Enter twice to continue or save, or Alt/Option-Q to return to the main menu.\n\
```
Comment (SwiftDust) [inline src/note.rs:42]: If I'm certain, that should work out of the box, why did you delete this line?

File: src/note.rs [hunk 2]
```diff
@@ -107,26 +123,52 @@ impl Note {
...
         };
 
-        if saved_notes.is_empty() {
-            cursor_to_origin()?;
-            println!("You can't edit notes, because there are none.");
-        }
-
         cursor_to_origin()?;
-        api::save_note(&updated_note, db_file)?; // why the fuck whas this line not here yet
+        api::save_note(&updated_note, db_file)?; // data in my tummy
```
Comment (SwiftDust) [inline src/note.rs]: Just remove this entire comment already...

File: src/note.rs [hunk 3]
```diff
@@ -57,10 +58,25 @@ impl Note {
...
             }
         }
+
+        println!("Content:");
+        let mut content = String::new();
+        loop {
+            let mut buffer = String::new();
+            io::stdin().read_line(&mut buffer)?;
+
+            // if nothing written then break
```
Comment (SwiftDust) [inline src/note.rs]: You don't really have to comment this...

**Review by SwiftDust — CHANGES_REQUESTED** [review]
Looks pretty nice, but I have a little feedback on some things.

* Firstly, you shouldn't be able to make an empty note (this is possible now).
* Could you maybe make it so that when the user presses Ctrl+Enter, it'll make a new line and when the user presses Enter, it ends the note?
* Could you make it so that the user can start typing on the same line as "Content:"?
* Could you make it so that when you edit a note, the content the user typed will be prefilled, but the user will be able to edit it?
* Could you make it so that the shortcuts work? (maybe something for another PR)

For the rest it's pretty nice! 
\- Mart

File: src/note.rs (hunk 1 above)
Comment (SwiftDust) [reply src/note.rs:42]: Okay, I tested it and it doesn't work, that's something to fix.

**Comment by ghost** [comment]
hey there @ThatFrogDev 

i was able to fix and add the
- empty note (fixed)
- same line as "Content: "

and for the other changes i will create a new PR soon

because they will probably require different plugins... hope you understand
